DataFormats/Phase2ITPixelCluster/src/Phase2ITPixelCluster.cc

```diff
@@ -0,0 +1,78 @@
+#include "DataFormats/Phase2ITPixelCluster/interface/Phase2ITPixelCluster.h"
+
+//---------------------------------------------------------------------------
+//!  \class Phase2ITPixelCluster
+//!  \brief Pixel cluster -- collection of pixels with ADC counts
+//!
+//!  Class to contain and store all the topological information of pixel clusters:
+//!  charge, global size, size and the barycenter in x and y
+//!  local directions. It builds a vector of SiPixel (which is
+//!  an inner class) and a container of channels. 
+//!
+//!  March 2007: Edge pixel methods moved to RectangularPixelTopology (V.Chiochia)
+//!  May   2008: Offset based packing (D.Fehling / A. Rizzi)  
+//!  \author Petar Maksimovic, JHU
+//---------------------------------------------------------------------------
+
+
+Phase2ITPixelCluster::Phase2ITPixelCluster( const Phase2ITPixelCluster::PixelPos& pix, uint32_t adc) :
+  thePixelRow(pix.row()),
+  thePixelCol(pix.col()),
+    // ggiurgiu@fnal.gov, 01/05/12
```

Comment:
@emiglior: There are still several false-history comments that should be deleted in the various files in this PR. Should I flag them all?